### PR TITLE
Add ffmpeg-full and update ffmpeg

### PR DESCRIPTION
**Pull request by VuiMuich**
## Description
- Adds package `ffmpeg-full` to install the `full build` of ffmpeg, whereas the `ffmpeg` package just installs the `essentials build`
- update descriptions of `ffmpeg` package to note that this is just installing the `essentials build`
- update the `update.ps1` script to use [this version info](https://www.gyan.dev/ffmpeg/builds/release-version) to check for the latest version (bugfix)

## Motivation and Context
For video heavy work environments it is helpful to have the full ffmpeg build with support for all possible codecs and features.
The maintainer of the remote decided to remove the build date from the version, so installing older versions with the same script is expected to fail.

## How Has this Been Tested?

- Set up this [chocolate-test-environment](https://github.com/majkinetor/chocolatey-test-environment)
- did test for install, uninstall and downgrade two versions back (that as much is provided by remote)
- Install/uninstall succeded
- `--version` fails, as expected

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect ~~a single package~~ two single packages (not including meta package).

### CODE REVIEW

File: automatic/ffmpeg-full/ffmpeg-full.nuspec
Comment (mwallner) [inline automatic/ffmpeg-full/ffmpeg-full.nuspec:6]: version 4.4 won't do here, as AU would fetch/set the this very version number, see "no change" and therefore do nothing after that (version wouldn't be pushed).

just put in `0.0`, that'd be fine

File: automatic/ffmpeg-full/ffmpeg-full.nuspec
Comment (mwallner) [inline automatic/ffmpeg-full/ffmpeg-full.nuspec:8]: you'll need to add yourself as package owner

File: automatic/ffmpeg/ffmpeg.nuspec
Comment (mwallner) [inline automatic/ffmpeg/ffmpeg.nuspec:6]: same as in the nuspec of ffmpeg-full, please revert to `4.3.2`

File: automatic/ffmpeg-full/ffmpeg-full.nuspec
Comment (mwallner) [inline automatic/ffmpeg-full/ffmpeg-full.nuspec:36]: is this really necessary?
(if it has been copied from the old package, please ditch this dependency)

**Review by mwallner — CHANGES_REQUESTED** [review]
thanks for working on this!
a few small changes and we can get it merged!

File: automatic/ffmpeg-full/ffmpeg-full.nuspec
Comment (VuiMuich) [reply automatic/ffmpeg-full/ffmpeg-full.nuspec:8]: Simply my Github Username? So like:
`<owners>chocolatey-community,VuiMuich</owners>`

**Review by VuiMuich — COMMENTED** [review]
After further lecture into nuspec I guess there is no such thing as "conflicting with package" in chocolatey right?

So I think this would be ready to merge from my side..

**Comment by VuiMuich** [comment]
@mwallner are we missing something and we still need some changes? or are we good to merge?

**Review by mwallner — APPROVED** [review]

**Comment by mwallner** [comment]
> @mwallner are we missing something and we still need some changes? or are we good to merge?
sorry - been quite busy lately - we're good to go imo.

**Comment by AdmiringWorm** [comment]
@VuiMuich what is your chocolatey username?
We prefer to have at least one other maintainer (the one considered the main maintainer) on the chocolatey-community packages when possible

**Comment by VuiMuich** [comment]
Is it possible to use the github handle? As I probably would check messages directly on chocolatey only every now and then as I use Windows almost exclusively at work.

**Comment by AdmiringWorm** [comment]
@VuiMuich unfortunately no, that isn't possible.
Chocolatey and GitHub aren't connected in that way, and a chocolatey user would be needed.

Not saying you have to respond to incoming messages/comments from users, it is more for discoverability. It is still recommended that users open an issue in this repository when they have problems instead.

If you don't have a chocolatey user, I am not going to force you to create one though.
I'll add you to the [CODEOWNERS](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/.github/CODEOWNERS) file so we have a reference on who to request a review from on any incoming pull requests.

**Comment by VuiMuich** [comment]
Ok, I just opened a chocolatey user `VuiMuich` and I'll try to give this attention at least once a week.

**Comment by AdmiringWorm** [comment]
@VuiMuich I have added you as a maintainer (pending your approval) to the package.

**Comment by VuiMuich** [comment]
Is a approving comment enough? Or do I approve somewhere else?

**Comment by AdmiringWorm** [comment]
@VuiMuich you should have received an email from chocolatey with a link to confirm being added as a maintainer

**Comment by VuiMuich** [comment]
That worked thanks, hadn't checked mails before as I am on mobile.

Btw: I noticed the title could be improved to `Ffmpeg Full`, should I use the `upload Package` function on chocolatey or open another PR?

**Comment by AdmiringWorm** [comment]
> Btw: I noticed the title could be improved to FFmpeg Full, should I use the upload Package function on chocolatey or open another PR?

Manual upload through the website do unfortunately not work, so it would need to be pushed from this repository (any change, be it for the metadata or the scripts requires a package fix version to show up on chocolatey.org).

> But the package site is not showing. Also there are reports of the package being broken, is this possibly due to the PR not being merged correctly or did I break something for ffmpeg?

Not seen any recent reports that it is broken myself, but I assume that there were no package fix version pushed to chocolatey.org which explains why your changes aren't in the package upstream.

**Comment by VuiMuich** [comment]
Oh, It appears @choco-bot removed 'line 24' in `b21ba68f5f03b8bc65d1cb84d583eddef5a27f02`
Any clues why it would do that? 
The hint that it is the essentials build is actually there, just didn't ready carefully enough, obviously.

There is [this comment in the discussion](http://disq.us/p/2gpu6ba) and one reply confirming, that it's broken:
```ps1
mpeg 4.4... 99% Progress: Downloading ffmpeg 4.4... 100%
ffmpeg v4.4 [Approved] ffmpeg package files upgrade completed. Performing other installation steps. Extracting 64-bit C:\ProgramData\chocolatey\lib\ffmpeg\tools\ffmpeg-release-essentials.7z to C:\ProgramData\chocolatey\lib\ffmpeg\tools... C:\ProgramData\chocolatey\lib\ffmpeg\tools
ERROR
: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to
Stop: Access to the path 'C:\ProgramData\chocolatey\lib\ffmpeg\tools\ffmpeg-4.4-essentials_build' is denied. The upgrade of ffmpeg was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\ffmpeg\tools\chocolateyInstall.ps1'.
See log for details. Chocolatey upgraded 0/1 packages. 1 packages failed.
See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Failures - ffmpeg (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\ffmpeg\tools\chocolateyInstall.ps1'.
See log for details.
```
I'll have a look at it next week..

**Comment by AdmiringWorm** [comment]
> Oh, It appears @choco-bot removed 'line 24' in b21ba68f5f03b8bc65d1cb84d583eddef5a27f02

Hmm, looks like the changes had been made in the nuspec directly, but not in the `Readme.md` file (during the update this file is read, and updates the nuspec from that file).

Regarding the error, the initial thought (without having looked into it) is that the user is running as a non-admin, or there is an executable running in the folder causing it to fail (seems unlikely in this case though, as the error should probably have happened earlier in the script).
More likely it is due to an anti-virus locking the folder due to either putting a file in quarantine, or it is scanning the folder.

**Pull request merged**
